src/sentry/static/sentry/app/routes.tsx

```diff
@@ -1659,11 +1682,11 @@ function routes() {
                 component={errorHandler(LazyLoad)}
               />
               <Redirect
-                path="new-events/"
+                from="new-events/"
```

Comment:
`path` wasn't in the types for `redirect`, but `from` is, which I verified in our specific version of react-router

https://github.com/ReactTraining/react-router/blob/42fe32d8f4ebec170a6d980badbc702ab88b5897/modules/Redirect.js#L24-L25

Is an alias.

This is a safe change.